javascriptv3/example_code/iam/actions/upload-server-certificate.js

```diff
@@ -13,40 +13,42 @@ import * as path from "path";
 
 const client = new IAMClient({});
 
-/**
- * The certificate body and private key were generated with the
- * following command.
- *
- * ```
- * openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes \
- * -keyout example.key -out example.crt -subj "/CN=example.com" \
- * -addext "subjectAltName=DNS:example.com,DNS:www.example.net,IP:10.0.0.1"
- * ```
- */
-
-const certBody = readFileSync(
-  path.join(
-    dirnameFromMetaUrl(import.meta.url),
-    "../../../../resources/sample_files/sample_cert.pem",
-  ),
-);
-
-const privateKey = readFileSync(
-  path.join(
-    dirnameFromMetaUrl(import.meta.url),
-    "../../../../resources/sample_files/sample_private_key.pem",
-  ),
-);
+const certMessage = `Generate a certificate and key with:
+
+openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes
```

Comment:
With this on multiple lines, copy/pasting this on any shell will run it as three commands. So an ending `\` will help there. But then, this is only valid for systems with `openssl` as a binary. Perhaps the wording should be genrally, "Do a <whatever> with this command (or equivalent for  your system)."